notebooks/output_files_tutorial.ipynb

```diff
@@ -1088,6 +1089,81 @@
     "## Part 4: Folded models and their ranking"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Folded model files (.pdb)\n",
+    "These files each contain one folded model. File names are formatted as {prefix}\\_{N ECs used for folding}\\_{model number}\\_hMIN.pdb\n",
+    "\n",
+    "### ECs with Secondary Structure Clashes (CouplingScores_with_clashes.csv):\n",
+    "Includes evolutionary couplings sorted by score, with a flag for potential secondary structure clashes (e.g. a coupling between two residues that are in the same helix that cannot be in contact directly due to the helix).  Columns are the same as \\CouplingScores.csv but with an additional column:\n",
+    "ss_clash - a boolean that is true if the coupling is a potential secondadry structure clash\n",
+    "\n",
+    "### Predicted Secondary Structures File (secondary_structure.csv):\n",
+    "Output predicted secondary structure for the protein, e.g. from Psipred as run in the standard version of the pipeline.  Columns are:\n",
+    "\n",
+    "- A_i: one-letter amino acid code for the residue of interest\n",
+    "- sec_struct_3state: one-letter code showing predicted secondary structure for that residue.  \n",
+    "        - H = helix, E = sheet, C = chain.\n",
+    "- sec_struct_conf: confidence in the secondary structure output, as given by Psipred.  9 is the most confident, 1 is the least.\n",
+    "- i: residue index\n",
+    "- in_model: a boolean flag that displays whether the residue has sufficient coverage in the \n",
+    "\talignment (and is therefore eligible to have evolutionary couplings identified)\n",
+    "\n",
+    "### Structure ranking file (ranking.csv):\n",
```

Comment:
add short sentence here that larger clusters usually tend to correspond to more reliable predictions (even if obvious)?